### DISCUSSION TITLE
What is this code collapsing thing and how do I disable it?

### DISCUSSION THREAD

**Discussion by squeaktoy** [LazyVim/LazyVim#4918] [Q&A]
![image](https://github.com/user-attachments/assets/5ae2479c-52d2-40c2-b4dd-ada411bbef7a)
The image should be self-explanatory. My code is collapsed and only expanded when I use `l`.
How can I disable this?

**Comment by guivictorr** [comment]
`zi` I think

<img width="213" alt="Screenshot 2025-04-19 at 16 07 43" src="https://github.com/user-attachments/assets/e48af351-9209-4f16-81be-cb47ce0e637f" />
